src/main/java/com/globalitgeeks/examninja/usermanagement/dto/CustomisedErrorResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.globalitgeeks.examninja.usermanagement.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
```

Comment:
We are not using ErrorResponse anywhere.. So it is better to delete that class and rename CustomisedErrorResonse as ErrorResponse

Reply:
Addressed 
We are using ErrorResponse for Method argumentsException